sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -1143,38 +1791,50 @@ class CompactBlockProcessor internal constructor(
      *
```

Comment:
This sets global parameters, rather than returning a new instance of `ProcessorInfo`, correct? Please make that clear in the documentation.